docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1037,6 +1038,20 @@ sequence. Therefore, do not use `second` in this case.
 
 end::inference-config-nlp-tokenization-truncate[]
 
+tag::inference-config-nlp-tokenization-truncate-deberta-v2[]
+Indicates how tokens are truncated when they exceed `max_sequence_length`.
+The default value is `first`.
++
+--
+* `balanced`: Both the first and second sequences are truncated such that `len(truncated(seq1)) = min(len(seq1), (max_sequence_length - special_character_count) / 2)` and `len(seq2) = max_sequence_length - len(truncated(seq1)) - special_character_count`.
```

Comment:
the semantics of this changed today so we need to change this line, let me know if this makes sense to you. 

```suggestion
* `balanced`: One or both of the first and second sequences may be truncated so as to balance the tokens included from both sequences
```